modules/apps/batch-engine/batch-engine-service/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
 	compileInclude group: "com.liferay", name: "poi-ooxml-lite", version: "5.2.2.LIFERAY-PATCHED-1"
 	compileInclude group: "com.liferay", name: "xmlbeans", version: "5.0.3.LIFERAY-PATCHED-1"
 	compileInclude group: "com.zaxxer", name: "SparseBitSet", version: "1.2"
+	compileInclude group: "commons-io", name: "commons-io", version: "2.11.0"
 	compileInclude group: "org.apache.commons", name: "commons-collections4", version: "4.3"
 
```

Comment:
@javalosjr96, if there is no process started in this module, you may use **compileOnly** because the lib had deployed as  module. You can see the lib org.apache.commons.commons-io.jar existed in liferay-home\osgi\static. I guess it was deployed by modules\apps\static\portal-osgi-web\portal-osgi-web-wab-extender\build.gradle.

In the commit message, please write the detailed explanation why the commit is necessary.